### PR TITLE
Comments upload export spec

### PR DESCRIPTION
**Pull request by aapomm**
- Add spec to test comments upload/export
- ~Remove comments' user validation~
- Add default user for upload

### CODE REVIEW

File: spec/features/export/v2/template.rb [hunk 1]
```diff
@@ -0,0 +1,28 @@
+require 'rails_helper'
+
+describe 'exporting comments' do
+  before { login_to_project_as_user }
+
+  context 'issue with a comment' do
+    before do
+      @issue = create(:issue, text: 'Sample issue')
+      @comment = @issue.comments.create(
```
Comment (dormi) [inline spec/features/export/v2/template.rb]: use factory?

File: spec/features/export/v2/template.rb [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+        content: 'Sample comment',
+        user_id: @logged_in_as.id
+      )
+    end
+
+    it 'creates the comment xml' do
+      export_options = {plugin: Dradis::Plugins::Projects}
+      @exporter =
```
Comment (dormi) [inline spec/features/export/v2/template.rb]: no '@' needed

File: spec/features/upload/v2/template.rb [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+
+    it "transforms methodologies into boards", js: true do
+      select "Dradis::Plugins::Projects::Upload::Template"
+      attach_file \
+        "file",
+        Rails.root.join(
+          "spec", "fixtures", "files", "templates", "with_comments.xml"
+        )
+
+      expect(page).to have_text("Worker process completed", wait: 120)
```
Comment (dormi) [inline spec/features/upload/v2/template.rb]: this `120` makes me thing about using something different from feature specs

Comment (aapomm) [reply spec/features/upload/v2/template.rb]: I just copied this from the `upload_spec` in Pro.

File: spec/features/export/v2/template.rb [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+      @issue = create(:issue, text: 'Sample issue')
+      @comment = create(:comment,
+        content: 'Sample comment',
+        commentable: @issue,
+        user: @logged_in_as
+      )
+    end
+
+    it 'creates the comment xml' do
+      export_options = {plugin: Dradis::Plugins::Projects}
```
Comment (etdsoft) [inline spec/features/export/v2/template.rb]: indent

File: spec/features/upload/v2/template.rb (hunk 3 above)
Comment (etdsoft) [reply spec/features/upload/v2/template.rb]: > I just copied this from the upload_spec in Pro.

Is that a good enough reason to keep it?

**Pull request merged**